File: patterns/toggle-pattern/rules-js/toggle-pattern.js
rules.JSRule({
  name: "Toggle Lampe",
  triggers: [triggers.ItemCommandTrigger("mqttSwitchIn2", "OFF")],
  execute: (event) => {
    if (items.getItem("modbusSwitchOut1").state !== "ON") {
      items.getItem("modbusSwitchOut1").sendCommand("ON");
    } else {
      items.getItem("modbusSwitchOut1").sendCommand("OFF");
    }
  }
});

File: patterns/toggle-pattern/rules-python/toggle_pattern.py
from openhab import rule, Registry
from openhab.triggers import ItemCommandTrigger


@rule(triggers=[ItemCommandTrigger("mqttSwitchIn2", "OFF")])
class ToggleLampe:
    def execute(self, module, input):
        lamp = Registry.getItem("modbusSwitchOut1")
        if str(lamp.getState()) != "ON":
            lamp.sendCommand("ON")
        else:
            lamp.sendCommand("OFF")
